hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/GatewayServiceEntityConverter.java

```diff
@@ -118,7 +130,30 @@ private BaselineEntity getBaselineEntity(
                 attrMap,
                 containerMap,
                 incomingEdges,
-                outgoingEdges));
+                outgoingEdges,
+                labelResultSet));
+  }
+
+  private LabelResultSet buildLabelResultSet(
+      org.hypertrace.gateway.service.v1.entity.Entity entity,
+      EntityRequest request,
+      Map<String, String> labelsMap) {
+    if (labelsMap.isEmpty()) {
+      return EMPTY_LABEL_RESULT_SET;
+    }
+    Value value =
+        entity.getAttributeOrDefault(request.labelsAttributeRequest().attribute().id(), null);
```

Comment:
Just wondering, should we use something like https://github.com/hypertrace/hypertrace-graphql/blob/1d9513e87d765ef4c549454c341193a2e92b037e/hypertrace-graphql-attribute-scope/src/main/java/org/hypertrace/graphql/atttribute/scopes/HypertraceAttributeScopeModule.java#L21, to map the foreign id keys between label id and label ids stored in entities?

A similar example case happens for tying up log event with a span https://github.com/hypertrace/hypertrace-core-graphql/blob/e6a3738f459a45bd45eb73ebd8f326de23dac2d0/hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/dao/SpanLogEventResponseConverter.java#L41

Reply:
as we discussed this is probably not needed